pycbc/workflow/core.py

```diff
@@ -764,6 +764,98 @@ def _filename(self, ifo, description, extension, segment):
         return "%s-%s-%s-%s.%s" % (ifo, description.upper(), start, 
                                    duration, extension)  
     
+class RemoteFile(File):
+    '''
```

Comment:
If the intention is for this to replace the File class, can you name it as such. It is not just for "remote" files. It should be agnostic. NewFIle would be good enough, or something else. I would be very unhappy if this class didn't replace the standard one in the next couple weeks.